pkg/monitortests/authentication/requiredsccmonitortests/monitortest.go

```diff
@@ -81,6 +86,17 @@ func (w *requiredSCCAnnotationChecker) CollectData(ctx context.Context, storageD
 
 			suggestedSCC := suggestSCC(&pod)
 			owners := ownerReferences(&pod)
+
+			// check if the SCC is non-standard and whether it's allowed in this namespace
+			validatedSCC := pod.Annotations[securityv1.ValidatedSCCAnnotation]
+			if _, isNonStandard := nonStandardSCCNamespaces[validatedSCC]; isNonStandard {
+				allowedNamespaces := nonStandardSCCNamespaces[validatedSCC]
```

Comment:
You can assign `allowedNamespaces` directly at the if statement, where `isNonStandard` is assigned as well.